src/materialized/bin/materialized.rs

```diff
@@ -114,6 +114,11 @@ fn run() -> Result<(), failure::Error> {
     );
     opts.optopt("", "symbiosis", "(internal use only)", "URL");
     opts.optflag("", "no-prometheus", "Do not gather prometheus metrics");
+    opts.optflag(
```

Comment:
I think the presence of this flag should be conditional on `!cfg(debug_assertions)`, so that folks running release builds don't need to worry about this at all.